awd/crossref.py

```diff
@@ -74,6 +74,38 @@ def create_dspace_metadata_from_dict(value_dict, metadata_mapping_path):
         return {"metadata": metadata}
 
 
+def is_valid_dspace_metadata(dspace_metadata):
+    """Validate that the metadata follows the format expected by DSpace."""
+    validation_status = False
+    approved_metadata_fields = [
+        "dc.contributor.author",
+        "dc.relation.journal",
+        "dc.identifier.issn",
+        "mit.journal.issue",
+        "dc.date.issued",
+        "dc.language",
+        "dc.title.alternative",
```

Comment:
Is 'dc.title.alternative' a multi value key (3 times)?

Reply:
A mistake from when I copied this over, it only needs to be there once